DESCRIPTION

```diff
@@ -40,14 +40,15 @@ Imports:
     shinyWidgets (>= 0.7.0),
     htmlwidgets,
     dplyr,
-    cohortBuilder (>= 0.2.0),
+    cohortBuilder (>= 0.2.2),
     tryCatchLog,
     highr,
     shinyGizmo (>= 0.4.2),
     rlang (>= 1.0),
     tibble,
     lifecycle
 Suggests:
+    shinyQueryBuilder(>= 0.1.0)
```

Comment:
As we have `shinyQueryBuilder` we should have the check if the package exists in `.gui_filter.query`. We should display the message that the package is required to use a query type filter.